extensions/ql-vscode/src/view/model-editor/ModeledMethodDataGrid.tsx

```diff
@@ -72,53 +68,56 @@ export const ModeledMethodDataGrid = ({
     : SINGLE_MODEL_GRID_TEMPLATE_COLUMNS;
 
   return (
-    <VSCodeDataGrid gridTemplateColumns={gridTemplateColumns}>
+    <DataGrid gridTemplateColumns={gridTemplateColumns}>
       {someMethodsAreVisible && (
         <>
-          <VSCodeDataGridRow rowType="header">
-            <VSCodeDataGridCell cellType="columnheader" gridColumn={1}>
-              API or method
-            </VSCodeDataGridCell>
-            <VSCodeDataGridCell cellType="columnheader" gridColumn={2}>
-              Model type
-            </VSCodeDataGridCell>
-            <VSCodeDataGridCell cellType="columnheader" gridColumn={3}>
-              Input
-            </VSCodeDataGridCell>
-            <VSCodeDataGridCell cellType="columnheader" gridColumn={4}>
-              Output
-            </VSCodeDataGridCell>
-            <VSCodeDataGridCell cellType="columnheader" gridColumn={5}>
-              Kind
-            </VSCodeDataGridCell>
-            {viewState.showMultipleModels && (
-              <VSCodeDataGridCell cellType="columnheader" gridColumn={6}>
-                <ScreenReaderOnly>Add or remove models</ScreenReaderOnly>
-              </VSCodeDataGridCell>
-            )}
-          </VSCodeDataGridRow>
-          {methodsWithModelability.map(({ method, methodCanBeModeled }) => {
-            const modeledMethods = modeledMethodsMap[method.signature] ?? [];
-            return (
-              <MethodRow
-                key={method.signature}
-                method={method}
-                methodCanBeModeled={methodCanBeModeled}
-                modeledMethods={modeledMethods}
-                methodIsUnsaved={modifiedSignatures.has(method.signature)}
-                modelingInProgress={inProgressMethods.has(method.signature)}
-                viewState={viewState}
-                revealedMethodSignature={revealedMethodSignature}
-                onChange={onChange}
-              />
-            );
-          })}
+          <DataGridCell gridRow={1} gridColumn={1}>
+            API or method
+          </DataGridCell>
+          <DataGridCell gridRow={1} gridColumn={2}>
+            Model type
+          </DataGridCell>
+          <DataGridCell gridRow={1} gridColumn={3}>
+            Input
+          </DataGridCell>
+          <DataGridCell gridRow={1} gridColumn={4}>
+            Output
+          </DataGridCell>
+          <DataGridCell gridRow={1} gridColumn={5}>
+            Kind
+          </DataGridCell>
+          {viewState.showMultipleModels && (
+            <DataGridCell gridRow={1} gridColumn={6}>
+              <ScreenReaderOnly>Add or remove models</ScreenReaderOnly>
+            </DataGridCell>
+          )}
```

Comment:
Can we add header styles to these like we had before? See https://github.com/microsoft/vscode-webview-ui-toolkit/blob/f22b1b0fab54bb41761388cb408be89335d9dc5c/src/data-grid/data-grid-cell.styles.ts#L41-L43

Reply:
Thanks for pointing this out. I did briefly look for special header styles, but I couldn't find any. I'll add this in.